src/ripe/order.py

```diff
@@ -324,11 +324,21 @@ def update_tag(self, number, identifier=None, type=None, activate=None):
         contents = self.put(url, identifier=identifier, type=type, activate=activate)
         return contents
 
+    def delete_tag(self, number):
+        url = self.base_url + "orders/%d/tag" % number
+        contents = self.delete(url)

```

Comment:
Add the deactivate kwarg that can be passed to delete_tag: https://github.com/ripe-tech/ripe-core/blob/master/src/ripe_core/controllers/api/order.py#L722

(similarly to the above update_tag method)